### PR TITLE
Template improvement

### PR DESCRIPTION
**Pull request by meets2tarun**
#1 
Test build [here](https://meilix-test.herokuapp.com/)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 Flask
 gunicorn
+gitpython
```
Comment (xeon-zolt) [inline requirements.txt]: we are not using gitpython its not working while deployment plus it downloads source each time 
i will change deployment script to download sources during deployment

Comment (xeon-zolt) [inline requirements.txt]: we are not using gitpython its not working while deployment plus it downloads source each time 
i will change deployment script to download sources during deployment

**Comment by meets2tarun** [comment]
I have edited the requirements.txt to remove that line 
Please review and squash and merge.

**Comment by xeon-zolt** [comment]
why layout file with unnecessary things ?

**Comment by meets2tarun** [comment]
Layout file contains the style elements and a tab in the bottom right.
See here the changes https://meilix-test.herokuapp.com/

**Comment by xeon-zolt** [comment]
lets merge it later when we have a working button till then we should hold this

**Comment by meets2tarun** [comment]
Let's follow a strategy to merge each other's PR.
No-one is allowed to merge its own PR unless there is a critical situation arises.
Try squash and merge button.
Thank You

**Pull request closed without merging**